src/WingetCreateCore/Common/PackageParser.cs

```diff
@@ -643,6 +643,37 @@ private static bool ParseExeInstallerType(string path, Installer baseInstaller,
             }
         }
 
+        /// <summary>
+        /// Checks if a MSI Installer database was generated by WiX, based on common characteristics.
+        /// </summary>
+        /// <param name="installer">A MSI Installer database.</param>
+        /// <returns>A boolean.</returns>
+        private static bool IsWix(QDatabase installer)
+        {
+            // Do any of the WiX properties exist?
+            foreach (var property in installer.Properties)
+            {
+                if (property.Property.ToLower().Contains("wix"))
+                {
+                    return true;
+                }
+            }
+
+            // Do any of the WiX tables exist?
+            foreach (var table in installer.Tables)
+            {
+                if (table.Name.ToLower().Contains("wix"))
+                {
+                    return true;
+                }
+            }
+
+            // Does the CreatingApp mention WiX?
+            return
+                installer.SummaryInfo.CreatingApp.ToLower().Contains("wix") ||
+                installer.SummaryInfo.CreatingApp.ToLower().Contains("windows installer xml");
```

Comment:
```suggestion
            // Do any of the WiX properties exist?
            // Do any of the WiX tables exist?
            // Does the CreatingApp mention WiX?
            return
                Array.Exists(installer.Properties, property => property.Property.ToLower().Contains("wix")) ||
                Array.Exists(installer.Tables, table => table.Name.ToLower().Contains("wix")) ||
                installer.SummaryInfo.CreatingApp.ToLower().Contains("wix") ||
                installer.SummaryInfo.CreatingApp.ToLower().Contains("windows installer xml");
```

Reply:
Had to modify your suggestions to convert them to Arrays (they didn't have Exist methods and apparently you can't just do IEnumerable.Exists on them either). Want to take a look?

Reply:
Replied in Gitter; May be able to use the `FirstOrDefault` method instead